exonum/src/node/events.rs

```diff
@@ -77,6 +77,10 @@ impl NodeHandler {
                 self.connect(&address);
             }
             ExternalMessage::Enable(value) => {
+                if self.node_role.is_auditor() {
+                    error!("Trying to enable consensus, but the current node is auditor and cannot affect consensus process");
```

Comment:
IMHO this message is too verbose. Wouldn't "Attempt to enable consensus for an auditor" be enough?

Reply:
Any reason *not* to make message verbose?

Reply:
By "verbose" I mean "using or expressed in more words than are needed". If we are able to write message in less words with the same amount of information, I'd prefer to do it.